audits/audit.js

```diff
@@ -41,12 +41,14 @@ class Audit {
   /**
    * @param {(boolean|number|string)} value
    * @param {?(boolean|number|string)=} rawValue
+   * @param {string=} debugString Optional string to describe any error condition encountered.
    * @return {!AuditResult}
    */
-  static generateAuditResult(value, rawValue) {
+  static generateAuditResult(value, rawValue, debugString) {
     return {
-      value: value,
-      rawValue: rawValue,
+      value,
```

Comment:
personally i really hate mixing the : syntax it is very confusing ... but maybe it is something to get used to for me.